website/versioned_docs/version-0.6.0/userDocs/yarn/docker/pytorch/with-cifar10-models/cifar10_tutorial.py

```diff
@@ -71,7 +71,12 @@
 
 Using ``torchvision``, it’s extremely easy to load CIFAR10.
 """
+import matplotlib.pyplot as plt
```

Comment:
These codes under 0.6.0 documents `website/versioned_docs/version-0.6.0/**` should also be rolled back.